.gitignore

```diff
@@ -2,4 +2,5 @@
 .idea
 .vscode
 .DS_Store
-.cache
\ No newline at end of file
+.cache
+json/
```

Comment:
Guessing it's to do with mkdocs?

Reply:
No CDK CLI JSON output option generates files.

Reply:
This is the top level `.gitignore` so I guess you ran cdk CLI accidentally there?

Reply:
good catch, I think I just clicked on the wrong gitignore. fixed now. moved to the awscdk project gitignore